Gitter/Gitter/Gitter.WindowsPhone/Services/WindowsPhoneNotificationService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.UI.Notifications;
+using Gitter.Configuration;
+using Gitter.Services.Abstract;
+
+namespace Gitter.Services.Concrete
+{
+    public class WindowsPhoneNotificationService : BaseNotificationService, ILocalNotificationService
+    {
+        public override void SendNotification(string title, string content, string id = null, string group = null)
+        {
+            var notification = this.CreateToastNotification(title, content, id);
+
+#if WINDOWS_PHONE_APP
```

Comment:
This was me attempting to get it to work under Windows Phone, but even if the compiler condition in place, it still wasn't finding the Tag and Group properties.  I have removed them for now.